bouncerscript/script.py

```diff
@@ -19,10 +24,10 @@
 async def bouncer_submission(context):
     """Implement the bouncer submission behavior"""
```

Comment:
The existing docstrings could use some work. The new functions could use docstrings.
I generally use [Google style docstrings](http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html). Adding them could be helpful for people working on bouncerscript, but doesn't block this PR.

Reply:
Good point. I'll add an item to my backlog to do this in all scripts I'm responsible for. Consisetency is good. Filed https://github.com/mozilla-releng/bouncerscript/issues/26 for now to track this, won't address it in this PR.